saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -218,6 +218,21 @@ class Meta:
         error_type_class = OrderError
         error_type_field = "order_errors"
 
+    @classmethod
+    def get_instance(cls, info, **data):
+        instance = super().get_instance(info, **data)
+        if instance.status != OrderStatus.DRAFT:
+            raise ValidationError(
+                {
+                    "id": ValidationError(
+                        "Provided order id belongs to non-order. "
```

Comment:
```suggestion
                        "Provided order id belongs to order instead of draft."
```
What do you think to improve description of this error?

Reply:
it was meant to be `non-draft` :sweat_smile: will change in a moment